.github/workflows/chromatic.yml

```diff
@@ -24,6 +24,6 @@ jobs:
       - name: Install dependencies
         run: pnpm install
       - name: Publish to Chromatic
-        env:
-          NODE_OPTIONS: "--max-old-space-size=8192"
```

Comment:
Will approve in case you are just looking to test the single blocking error

Saw the env var wasn't added, but it looks like it has support for it, so checking that was intentional -- it will likely fail on build without it. https://www.chromatic.com/docs/github-actions/#support-for-environment-variables

Also wondering if node won't be present for some reason why they added the node setup in their basic flow we looked at yesterday: https://www.chromatic.com/docs/github-actions/#workflow-setup

Reply:
I noticed that node setup action too! Yeah I wanted to keep it as simple as possible and then go from there adding extra stuff only if needed